src/main/java/ca/corefacility/bioinformatics/irida/repositories/filesystem/IridaFileStorageUtility.java

```diff
@@ -106,4 +121,26 @@
 	 * @throws IOException if the files have different or invalid extensions
 	 */
 	public String getFileExtension(List<? extends SequencingObject> sequencingObjects) throws IOException;
+
+	/**
+	 * Delete local temporary file and/or directory.
+	 *
+	 * @param filePath The {@link Path} to the file
+	 * @param directoryPath The {@link Path} to the directory which has the file
+	 */
+	public static void cleanupLocalTemporaryFiles(Path filePath, Path directoryPath) {
```

Comment:
After reverting back to `Files.deleteIfExists` in those places we discussed, you should be able to remove this `static` method.  I think it should likely go away.

Reply:
Removed in 1dbdb47af3eedeee57410916380b562e92a85c41